fairlearn/metrics_plots/plot_disparities_in_selection_rate.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+from fairlearn.metrics import selection_rate_group_summary
+
+
+def plot_disparities_in_selection_rate(y_true, y_pred, sensitive_features):
+    """Plot disparities in selection for a binary classifier with a single binary sensitive feature.
```

Comment:
Same here. Looks like this should generalize beyond binary sensitive features

Reply:
thanks!